src/main/java/in/twizmwaz/cardinal/module/modules/teamManager/TeamManagerModule.java

```diff
@@ -101,7 +101,13 @@ private void removePlayer(Player player) {
                 team.remove(player);
             }
         }
+        this.clearHeldAttribute(player);
         Bukkit.getServer().getPluginManager().callEvent(new ScoreboardUpdateEvent());
     }
 
+    private void clearHeldAttribute(Player player) {
```

Comment:
This method removes the attribute modifiers of the player's held `ItemStack` from the, just before they quit.

Without this, the player's attributes, modified by said attribute modifiers, will be serialized. Normally, this isn't a problem, as the player will spawn holding the same item on their next join.

However, because players spawn in as Observers, lacking the item with the attribute modifiers, the modified attribute will be applied, without any way to remove it by switching items.